credentials/github.toml

```diff
@@ -0,0 +1,2 @@
+api_token = 
```

Comment:
Don't commit this file to the repo - in fact, I'm pretty sure this entire folder should be in `.gitignore`? If not, it should be added.

Reply:
Since it was the first time this file was made, I thought it would need to be added at least once. How do i remove this file?

Reply:
Ah, makes sense. Yeah, we don't want it to be added, just in case somebody accidentally commits credentials. If you remove the file, git should have a "file deleted" change - if you commit that, it should be removed from the repo (and, since I did check and this directory is in `.gitignore`, it shouldn't be added again even if you change it later).